guides/common/modules/proc_restricting-custom-repositories-to-a-specific-os-version-or-architecture.adoc

```diff
@@ -0,0 +1,20 @@
+[id="Restricting_Custom_Repositories_to_a_Specific_OS_Version_{context}"]
+= Restricting Custom Repositories to a Specific Operating System or Architecture in {Project}
+
+You can configure {Project} to make custom repositories available only on hosts with a specific operating system version or architecture.
+For example, you can restrict custom repositories only to {RHEL}{nbsp}9 hosts.
+
+ifdef::satellite[]
+[NOTE]
+Only restrict architecture and operating system version for custom products.
+{Project} applies these restrictions automatically for Red{nbsp}Hat repositories.
+endif::[]
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Click the product that contains the repository sets you want to restrict.
+. In the *Repositories* tab, click the repository you want to restrict.
+. Navigate to *Publishing Settings* > *Restrict to OS version* to restrict the operating system version.
++
+Navigate to *Publishing Settings* > *Restrict to architecture* to restrict the architecture.
+. Click *Save* to submit your changes.
```

Comment:
```suggestion
```

I'd leave it out because the user has to click "Save" after selecting an item from the list in the Web UI. There is no other "final ACK" that the user has to do.